aten/src/ATen/core/op_registration/op_registration.h

```diff
@@ -221,17 +221,17 @@ class CAFFE2_API RegisterOperators final {
      * >         .schema("my_op")
      * >         .kernel<decltype(my_kernel_cpu), &my_kernel_cpu>(TensorTypeId::CPUTensorId));
      */
-    template<class FuncType, FuncType* kernel_func>
+    template<class FuncType>
```

Comment:
This is public API, don't change it. Also I don't think you get any binary size improvements from changing this one.

Reply:
Correction: this one might actually give binary size benefits, it's used for a few ops from native_functions.yaml. Because of the public API thing, we should probably introduce a new variant instead of changing this one though. We can remove these after going through a deprecation cycle if we want to.

Reply:
Correction: This is used for some ops in native_functions.yaml and might actually hurt binary size. Because of the public API thing, instead of changing this, let's just not delete the overloads below and use them instead.